Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -5260,7 +5260,7 @@ ExceptionOr<Ref<SerializedScriptValue>> SerializedScriptValue::create(JSGlobalOb
         if (auto arrayBuffer = toPossiblySharedArrayBuffer(vm, transferable.get())) {
             if (arrayBuffer->isDetached() || arrayBuffer->isShared())
                 return Exception { ExceptionCode::DataCloneError };
-            if (arrayBuffer->isLocked()) {
+            if (!arrayBuffer->isDetachable()) {
```

Comment:
Alternative to this change is calling `pinAndLock` and then adding an `unlock` to ArrayBuffer. Then change to SerializedScriptValue.cpp can be reverted.

Any opinions on which approach to take @WebKit/jsc-reviewers ?

Reply:
I think this `isDetachable` looks fine.